### PR TITLE
feat: alias getText to get text attribute

### PR DESCRIPTION
**Pull request by jlipps**

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]

**Comment by jlipps** [comment]
OK finally had a chance to test this on a real device. It works, but in the process I had to update the test app, which found some bugs in the node focusing algorithm, so I fixed all those in this PR too.

File: package.json [hunk 1]
```diff
@@ -61,12 +61,13 @@
     "singleQuote": true
   },
   "dependencies": {
-    "@appium/types": "0.5.0",
+    "@appium/types": "0.14.2",
+    "@tsconfig/node14": "14.1.0",
     "@xmldom/xmldom": "0.8.10",
+    "appium": "2.2.2",
```
Comment (KazuCocoa) [inline package.json:67]: Should we have appium dependence here?

Comment (jlipps) [reply package.json:67]: no that seems like a mistake

**Review by KazuCocoa — APPROVED** [review]

**Comment by jlipps** [comment]
ok published in 2.3.0

**Pull request merged**